src/ProductBundle/Admin/ProductAdmin.php

```diff
@@ -80,22 +66,6 @@ public function getProductType()
         return $this->getRequest()->get('provider');
     }
 
-    /**
-     * Returns the product class from the provided request.
-     *
-     * @return string
-     */
-    public function getProductClass()
```

Comment:
Not sure if we can allow this kind of BC break. Or we can allow it. The other way is just to deprecate the method

Reply:
Sure, lets deprecate it.